### ISSUE TITLE
Generating font returns "save failed" despite "passed validation"

### ISSUE THREAD

**Issue by Darragoth** [fontforge/fontforge#5504]
### When reporting a bug/issue:
- [x] Screenshot
![FontForge](https://github.com/user-attachments/assets/40247c07-af1d-4e3c-a0ba-e83fd2fa5860)

- [x] The FontForge version and the operating system you're using
FontForge Win10 build (32-bit) Sun, Jan 1, 2023.
- [x] The behavior you expect to see, and the actual behavior
Expectations included generating a modestly modified Roboto Regular .ttf font file
- [ ] Steps to reproduce the behavior
- [ ] \(optional) Possible solution/fix/workaround

### When you open an issue for a change/improvement/feature request:
- [x] A description of the problem you're trying to solve, including _why_ you think this is a problem
FontForge fails to generate a ttf. font file
[Roboto-Regular-Jwerch.zip](https://github.com/user-attachments/files/17992483/Roboto-Regular-Jwerch.zip)
 despite every glyph passing the validation. Selecting "Validate before saving" before generating the file returns "Errors detected; Self Intersecting", but the "Review" returns immediately "Validation passed", while "Generate" returns "Save failed" and only those. 
- [ ] If the feature changes current behavior, reasons why your solution is better
- [ ] \(optional) Possible solution/fix/workaround

**Comment by Bry10022** [comment]
The Save Failed message can occur if the font file is open in the Windows font viewer, or when trying to save to a location that the user is not allowed to save to.

**Comment by Darragoth** [comment]
> The Save Failed message can occur if the font file is open in the Windows font viewer, or when trying to save to a location that the user is not allowed to save to.

Indeed, saving into another location solved the issue. Too bad FontForge doesn't suggest such.
